funnel/views/profile.py

```diff
@@ -242,6 +243,61 @@ def view(self) -> ReturnRenderWith:
 
         return ctx
 
+    def jsonify_calendar(jsonData):
+        return jsonify(jsonData['projects'])
+
+    @route('calendar', methods=['GET'], endpoint='calendar')
+    @requestargs('start', 'end')
+    @render_with(
+        {
+            'text/html': 'profile_calendar.html.jinja2',
+            'application/json': jsonify_calendar,
+        }
+    )
+    def org_calendar(
+        self, start: datetime = None, end: datetime = None
+    ) -> ReturnRenderWith:
+        projects = []
+        if start is not None and end is not None:
+            start = datetime.strptime(start, "%Y-%m-%dT%H:%M:%S%z")
+            end = datetime.strptime(end, "%Y-%m-%dT%H:%M:%S%z")
```

Comment:
The values were declared as datetime, but here they are assumed to be strings. Also, these calls can fail with bad data. Use `parse_isoformat` instead of `strptime`.